.travis/is-bump-commit.sh

```diff
@@ -2,7 +2,7 @@
 
 isBumpCommit() {
   msg=`git log --pretty=format:'%s' -1`
-  if [ "$msg" = "Automated version bump" ]
+  if [[ "$msg" =~ ^\[pr-bumper\]* ]]
```

Comment:
Ah, somehow I overlook the square brackets. Yeah I see no reason to ever do that manually in a commit message.